python/mxnet/gluon/loss.py

```diff
@@ -409,6 +409,8 @@ class CTCLoss(Loss):
         length respectively.
     weight : float or None
         Global scalar weight for loss.
+    blank_label : {'first', 'last'}, default 'last'
```

Comment:
it was intentional not to expose this option in gluon.

Reply:
That means I need to revert this commit and  resend a pull request ?

Reply:
you can simply add a commit that removes the change in blank_label